dev

```diff
@@ -0,0 +1,6 @@
+"docker logs" requires exactly 1 argument.
```

Comment:
This file seems to be non-needed and appears to be output from a docker-command piped wrongly maye?

Reply:
`make docker-logs` shows the logs from within a running container. Since no containers are running during the test, the command fails.

Reply:
I gave each container type a unique name since there may be multiples running at once. EG freqtrade-backtest, freqtrade-bot, freqtrade-shell. That means that `make logs-freqtrade-backtest`, is just as annoying as `docker logs freqtrade-backtest` so I removed the target.

Reply:
I think you miss-read my comment - i think the file "dev" is not necessary as it only contains the output of `docker logs`.

Reply:
Ah yes. I see. I deleted the file.